### DISCUSSION TITLE
when an update for this great project?

### DISCUSSION THREAD

**Discussion by hafbtm** [hansschmucker/NVStreamer1080#22] [General]
I would like to know if this project will continue to be updated in the future, mainly for greater compatibility with sunshine. I think it is a great tool for those of us who do not have much knowledge of commands and prefer a graphical interface.

**Comment by hansschmucker** [comment]
Hi hafbtm
To be honest, I'm not quite sure. After setting up Sunshine properly I've stopped using it myself, but... it's not entirely out of the question. But I'm not sure how it will continue.
Could be as a frontend for Sunshine config, could be in it's current form, could be not at all... I'm not entirely sure myself.

Currently, the second form (automating what Sunshine already offers) seems the most likely, but I'm not sure when I'll get around to that either, sorry.
